ambry-utils/src/main/java/com.github.ambry.utils/Utils.java

```diff
@@ -732,6 +741,16 @@ public static Throwable getRootCause(Throwable t) {
   }
 
   /**
+   * Get time in ms to the nearest second(floored) for the given time in ms
```

Comment:
I found the documentation and naming of this function quite confusing. Hard to understand what it is doing without looking at the code.